service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwin.java

```diff
@@ -246,34 +246,6 @@ public synchronized void updateTwin(DeviceTwinDevice device) throws IotHubExcept
                 proxy);
     }
 
-    /**
-     * This method replaces desired properties for the specified device. desired properties can be input
-     * via device's
-     * {@link com.microsoft.azure.sdk.iot.service.devicetwin.DeviceTwinDevice#setDesiredProperties(TwinCollection)}
-     * method.
-     *
-     * @param device The device with a valid id for which device twin is to be updated.
-     * @throws UnsupportedOperationException This exception is always thrown.
-     */
-    public void replaceDesiredProperties(DeviceTwinDevice device) throws UnsupportedOperationException
-    {
-        // Currently this is not supported by service
-        throw new UnsupportedOperationException();
```

Comment:
I missed this when removing deprecated APIs, but since we never implemented these methods, now is a good time to remove them